Library/Homebrew/brew.sh

```diff
@@ -150,72 +154,73 @@ update-preinstall-timer() {
   echo 'Updating Homebrew...' >&2
 }
 
+# shellcheck disable=SC2154
 update-preinstall() {
-  [[ -z "$HOMEBREW_HELP" ]] || return
-  [[ -z "$HOMEBREW_NO_AUTO_UPDATE" ]] || return
-  [[ -z "$HOMEBREW_AUTO_UPDATING" ]] || return
-  [[ -z "$HOMEBREW_UPDATE_PREINSTALL" ]] || return
-  [[ -z "$HOMEBREW_AUTO_UPDATE_CHECKED" ]] || return
+  [[ -z "${HOMEBREW_HELP}" ]] || return
+  [[ -z "${HOMEBREW_NO_AUTO_UPDATE}" ]] || return
+  [[ -z "${HOMEBREW_AUTO_UPDATING}" ]] || return
+  [[ -z "${HOMEBREW_UPDATE_PREINSTALL}" ]] || return
+  [[ -z "${HOMEBREW_AUTO_UPDATE_CHECKED}" ]] || return
 
   # If we've checked for updates, we don't need to check again.
   export HOMEBREW_AUTO_UPDATE_CHECKED="1"
 
-  if [[ "$HOMEBREW_COMMAND" = "cask" ]]
+  if [[ "${HOMEBREW_COMMAND}" = "cask" ]]
   then
-    if [[ "$HOMEBREW_CASK_COMMAND" != "upgrade" && $HOMEBREW_ARG_COUNT -lt 3 ]]
+    if [[ "${HOMEBREW_CASK_COMMAND}" != "upgrade" && ${HOMEBREW_ARG_COUNT} -lt 3 ]]
```

Comment:
Can we delete `HOMEBREW_CASK_COMMAND` because `brew cask` command was deprecated ?

Reply:
Yes, sounds good! Also this `[[ "${HOMEBREW_COMMAND}" = "cask" ]]`